v2/go.mod

```diff
@@ -1,59 +1,35 @@
 module github.com/estesp/manifest-tool/v2
 
-go 1.17
+go 1.18
 
 require (
 	github.com/containerd/containerd v1.6.16
 	github.com/docker/distribution v2.8.1+incompatible
-	github.com/docker/docker v20.10.17+incompatible
+	github.com/docker/docker v20.10.23+incompatible
 	github.com/fatih/color v1.10.0
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.1.0-rc2
 	github.com/pkg/errors v0.9.1
-	github.com/sirupsen/logrus v1.9.0
-	github.com/urfave/cli v1.22.12
-	gopkg.in/yaml.v2 v2.4.0
-	oras.land/oras-go v1.2.0
+	github.com/sirupsen/logrus v1.8.1
+	github.com/urfave/cli v1.22.9
```

Comment:
should be able to keep the upgrade of logrus and urfave/cli with your PR; everything else looks fine

Reply:
Ah, did I mess it up? That often happens when dealing with rebasing go packages. I will fix it right away. Hang on.

Reply:
OK, think I got it. Take another look?

Reply:
yup, thanks!